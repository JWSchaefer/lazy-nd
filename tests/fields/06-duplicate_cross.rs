use lazy_nd::lazy_nd;
#[lazy_nd(dim = D)]
struct Test<const D: usize> {
    #[scalar(mass:f64)]
    #[scalar(position : f64)]
    mass: f64,
}
fn main() {}
