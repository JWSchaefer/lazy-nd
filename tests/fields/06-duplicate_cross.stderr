error: field `mass` already declared
 --> $DIR/06-duplicate_cross.rs:6:5
  |
6 |     mass: f64,
  |     ^^^^
